.storybook/src/accessibility-tool.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useCallback} from 'react'
+import {useGlobals} from '@storybook/api'
+import {IconButton, ThemeProvider, Tooltip} from '../../src'
+import {AccessibilityIcon} from '@primer/octicons-react'
+
+export const ADDON_ID = 'myaddon'
+export const TOOL_ID = `${ADDON_ID}/tool`
+
+export const Tool = () => {
+  const [{showSurroundingElements, colorScheme}, updateGlobals] = useGlobals()
+
+  const toggleMyTool = useCallback(() => {
+    const currentValue = showSurroundingElements ?? window?.localStorage.getItem('showSurroundingElements') === 'true'
+
+    window?.localStorage?.setItem('showSurroundingElements', `${!currentValue}`)
+
+    updateGlobals({
+      showSurroundingElements: !currentValue
+    })
+  }, [showSurroundingElements])
+
+  return (
+    <ThemeProvider colorMode="day" dayScheme={colorScheme}>
```

Comment:
Per my suggestion on the issue, this is the code I played around with on your branch

```
<Box display="flex" alignItems="center" gap="4px">
          <Box flexGrow={1} fontSize={0} color="black" fontWeight="bold" id="switchLabel">
            Wrap links
          </Box>
          <ToggleSwitch aria-labelledby="switchLabel" size="small" />
        </Box>
```